src/test/ui/diverging-tuple-parts-39485.stderr

```diff
@@ -12,8 +12,10 @@ LL | fn g() -> &_ {
    |        ^^^^^
 help: consider removing the borrow
    |
-LL |     panic!()
-   |     ^^^^^^^^
+LL |     {
+LL |         $crate::panic!("explicit panic")
+LL |     }
+   |
```

Comment:
Ohh, I think I see what's happening, it's looking at the macro span now instead of the invocation span.

Maybe it's using some other heuristic we didn't think of yet?